### DISCUSSION TITLE
downloading folder as ZIP results in an error

### DISCUSSION THREAD

**Discussion by dprinzensteiner** [Studio-42/elFinder#3380] [Show and tell]
I have the elfinder installed via composer, and when I try to download a single file, it works fine. But when i try to download a folder, the zip archive is corrupted.
During debugging i found out, that after the zipdl function the outputJson function is called, in which the headers are set to application/json.

Has anyone encountered this issue before or do I have to configure some setting to enable the zipdl properly?

**Comment by ZerxaFun** [comment]
Up.
